src/docs/product/releases/health/index.mdx

```diff
@@ -0,0 +1,62 @@
+  - /workflow/releases/health/
+  - /product/releases/health/
+description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
+---
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session-data). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+## Enabling Release Health
+
+Many SDKs automatically manage the start and end of sessions when the SDK is initialized, but release health configuration is key to ensuring you are receiving data that is useful, so check our information on [which SDKs support release health](/product/releases/health/setup/) for links to configuring the SDK, as needed.
+
+## Data Filtering
+
+Sentry provides [inbound data filters](/product/error-monitoring/filtering/) to filter data into your project. Data filters are only partially supported for session events.
+
+## What Release Health Monitors
+
+Once you configure your SDK, Sentry connects the metrics to the specific release of your application and the associated code.
+
+### Session Data
+
+The primary component Sentry uses to monitor health is a _session_, which represents the interaction between the user and the application.
+
+A session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground.
+
+A session ends with the closing of the application or with the application being sent to the background. If the application is in the background for less than 30 seconds, we do not start the session again. Applications that are active even on the background (for example, a music player) should track the sessions manually for the background process.
+
+Sessions are submitted to Sentry so you can track the usage and adoption of your application. When a user of your application experiences a crash, error, or abnormal exit, the session will be flagged accordingly, and Sentry calculates derived metrics. The metrics include data such as the number of users that didn't experience a crash in the specified time range.
+
+### Active Users
+
+Number of users that started the application at least once in the last 24 hours.
+
+### Crash
+
+The fatal error that caused the crash of the application. Errors that did not cause the end of the application should not be included.
+
+Sentry considers every error event with the field `handled` set to `false` as a Crashed state.
+
+### Crash Free User
+
+Percentage of users who did not experience a crash during the specified time period.
+
+### Crash Free Sessions
+
+Number of sessions in the specified time range that did not end by a crash of the application.
+
+### Crashed Users
+
+Number of users that experienced a crash in the specified time range.
```

Comment:
This is actually confusing now. Why is "Crash Free User" a percentage and "Crashed Users" a count? I'll have to check if we really use it like this in the product, but there's some inconsistency.

Reply:
I'll combine, as makes sense. We will need to review these definitions for accuracy.

Reply:
Added a task to the list in #2869.